Script/Source/Entities/Entity.ts

```diff
@@ -60,6 +65,15 @@ namespace Script {
             this.currentHealth = this.health;
             this.position = _pos;
 
+            //move stuff
+            this.moved = false;
+            if (this.select(this.moves, true)[0]) {
```

Comment:
theoretisch sollte der zweite Parameter hier `false` sein, ist aber in unserem Fall egal... Vorallem weil wir den Code in zwei Wochen eh wegwerfen und von vorne anfangen xD